playwright/data/status-codes.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
For me this whole object is unnecessary.
I would suggest to just give the value `StatusCodes.NOT_FOUND` (which is 404) as the modified status code in the `response-modification.spec.ts` before part and then expecting that, and maybe `StatusCodes.OK` to not be in the response later on?
(and also change back in `response-validation.spec.ts`).

Reply:
You are right, I've changed it back, but I'll have a related question next time we have a meeting :)